build.mk

```diff
@@ -14,13 +14,22 @@ ifeq (${SGX_MODE}, HW)
 endif
 GO := $(GO_CMD) $(GOFLAGS)
 
+# in CI build, enable code coverage; disable it otherwise
+ifeq (${IS_CI_RUNNING}, true)
+	#this enables coverage in c/cpp code, libraries and linked binaries
+	export CODE_COVERAGE_ENABLED ?= true
+else
+	export CODE_COVERAGE_ENABLED ?= false
+endif
```

Comment:
Would that also fit into `config.mk` where also other exports are?

Reply:
I moved it there